_includes/pbc/all/install-features/latest/install-the-ssp-asset-management-feature.md

```diff
@@ -349,21 +382,21 @@ Make sure glossary keys have been added to `spy_glossary_key` and `spy_glossary_
 
 ## Set up behavior
```

Comment:
Is it behavior?

Reply:
Yes, usually plugins are descrined in the following section.